app/models/ems_refresh/save_inventory_infra.rb

```diff
@@ -300,7 +310,7 @@ def save_storage_profile_storages_inventory(storage_profile, storages)
     end
 
     save_inventory_multi(storage_profile.storage_profile_storages, hashes,
-                         :use_association, [:storage_profile_id, :storage_id])
+                         [], [:storage_profile_id, :storage_id])
```

Comment:
Both the [storage](https://github.com/ManageIQ/manageiq/blob/master/app/models/storage.rb#L7) and [storage_profile](https://github.com/ManageIQ/manageiq/blob/master/app/models/storage_profile.rb#L3) associations have `:dependent => :destroy` so I think it is safe to let the delete of either of those handle deleting this (it passes [this test](https://github.com/ManageIQ/manageiq/blob/master/spec/models/manageiq/providers/vmware/infra_manager/refresher_storage_profile_spec.rb#L36)) but I'd like someone to double check this, or maybe there is a better way?
This is so we don't have to return all the `storage_profile_storages` in targeted refresh